trompace/connection.py

```diff
@@ -13,6 +13,23 @@
     """
     q = {"query": querystr}
     server = config.server_id
+    print(server)
```

Comment:
be careful about completely copying code. In this case I would have got rid of these unnecessary print messages. We should also improve the error reporting in this case - this method should raise an Exception in the case of an error so that the calling client can make a decision on what to do

Reply:
Done, but I believe this will be shifted to the trompa processing library.